.github/actions/setup-env/action.yml

```diff
@@ -5,7 +5,7 @@ inputs:
   pnpm-version:
     description: Version of pnpm to install
     required: false
-    default: "9"
+    default: "10.17.1"
```

Comment:
note: i don't think this need to be aligned with the vesion in the package.json, but it has to be a recent (10.something).

pnpm will see the package.json and download the right version automatically

Reply:
Apparently, we can just omit the version from the `pnpm/action-setup` and it will take the version specified in the `packageManager` field: https://github.com/pnpm/action-setup?tab=readme-ov-file#install-only-pnpm-with-packagemanager

I have applied that change.